scripts/vagrant-setup.sh

```diff
@@ -65,7 +65,7 @@ if [ -d pulp-smash ]; then
   },
   "systems": [
     {
-      "hostname": "https://$(hostname)",
+      "hostname": "$(hostname)",
```

Comment:
If you'd like to excactly match the hostname that Pulp Smash is comparing against, try using this:

    "$(python -c 'import socket; print(socket.getfqdn())')"

Reply:
I'm not planning on changing this if only because it already works correctly.